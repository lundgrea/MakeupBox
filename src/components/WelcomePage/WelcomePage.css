.welcome-page {
  text-align: center;
  /* border: 2px solid grey; */
  border-radius: 6px;
  margin: 50px 400px 0px 400px;
  padding: 20px 10px 20px 10px;
  color: #242B28;
  background-color: #8EA99E;
} 

.welcome-header {
  font-size: 40px;
  margin: 10px 0px 5px 0px;
}

.welcome-CTA {
  margin: 40px 60px 10px 60px;
}

.welcome-experience {
  margin: 40px 0px 10px 0px;
}

.button-start-quiz {
  margin: 30px 10px 10px 10px;
}