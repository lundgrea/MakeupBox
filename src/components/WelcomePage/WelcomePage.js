import React from 'react';
import {Link} from 'react-router-dom'
import './WelcomePage.css'

export const WelcomePage = () => {
  return (
    <section className="welcome-page">
      <h2 className="welcome-header">Welcome to TAKEUP</h2>
      <h3 className="welcome-subhead">A guided makeup exploration experience built just for men interested in taking up makeup.</h3>
      <h4 className="welcome-experience">No experience necessary.</h4>
      <p className="welcome-CTA">To start, help us get to know you. Complete the short quiz so we can get you on your way.</p>
      <Link to='/quiz'><button className='button-start-quiz'>START QUIZ</button></Link>
    </section>
  )
}

export default WelcomePage;