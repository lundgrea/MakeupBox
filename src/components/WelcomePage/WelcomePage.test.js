import React from 'react';
import { shallow } from 'enzyme';
import WelcomePage from './WelcomePage';

describe('WelcomePage', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<WelcomePage />);
    expect(wrapper).toMatchSnapshot();
  });
});