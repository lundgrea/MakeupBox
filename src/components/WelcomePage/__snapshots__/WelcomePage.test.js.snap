// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomePage should match the snapshot 1`] = `
<section>
  <h2>
    Welcome to makeupBOX
  </h2>
  <h3>
    no experience required
  </h3>
  <p>
    First, help us get to know you.
  </p>
  <Link
    className="button-start-quiz"
    to="/quiz"
  >
    <button>
      START QUIZ
    </button>
  </Link>
</section>
`;
